src/helpers/validateId.js

```diff
@@ -0,0 +1 @@
+export const isId = str => /^@[a-zA-Z\/0-9+]+=.ed25519$/g.test(str)
```

Comment:
probably use [this regex](https://github.com/ssbc/ssb-ref/blob/master/index.js#L7) to be on the safe side

Reply:
done